Sprintfinity3902/States/ItemPickup/FairyPickup.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Sprintfinity3902.Link;
+using Sprintfinity3902.Interfaces;
+
+namespace Sprintfinity3902.Entities
+{
+    public class FairyPickup : IPickup
+    {
+        public FairyPickup()
+        { 
+
+        }
+
+        public bool Pickup(Player Link)
+        {
+            Link.LinkHealth = Link.MaxHealth;
+            Link.heartChanged = true;
+
+            Sound.SoundLoader.Instance.GetSound(Sound.SoundLoader.Sounds.LOZ_Get_Heart).Play(Global.Var.VOLUME, Global.Var.PITCH, Global.Var.PAN);
```

Comment:
I am not sure if the Fairy has a specific sound associated.

Reply:
Removing it now!